javascript/sdk/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Prelude SDK</title>
+  </head>
+  <body>
+    <script type="module">
+      import * as Prelude from "/lib/main.ts";
+      import * as uuid from "https://jspm.dev/uuid";
+
+      const service = new Prelude.Service({
+        host: "https://detect.dev.prelude.org",
+      });
+
+      const credentials = await service.iam.newAccount("test@example.com");
+
+      service.setCredentials(credentials);
+
+      const manifests = await service.build.listManifest();
+
+      console.log("Manifests before create", manifests);
+
+      const ttpId = uuid.v4();
+      await service.build.createTTP(ttpId, "This is a test");
+
+      const manifests2 = await service.build.listManifest();
+
+      console.log("Manifests after create", manifests2);
```

Comment:
anywhere you use 'manifests' swap to 'manifest' for proper usage

Reply:
Do you get the manifest that was just created?

Reply:
Got it